portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java

```diff
@@ -30,8 +30,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/chart")
-public class ChartServlet extends HttpServlet {
+@WebServlet("/numcomment-chart")
+public class NumCommentChartServlet extends HttpServlet {
 
   /*
    * Called when a client submits a POST request to the /chart URL
```

Comment:
nit: s/chart/numcomment-chart

Reply:
Did you mean replacing /chart with /numcomment-chart in the comment?

Reply:
yup that's what I mean!